### PR TITLE
chore: update README

### PR DESCRIPTION
**Pull request by k3llymariee**
very much a WIP, please provide lots of feedback unless this is Good Enough For Now.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,69 @@
...
-Enable a flag with key `test-flag` in project `default` and environment `production`.
-```
-go run main.go flags update \
-    --access-token {access-token} \
-    --base-uri http://localhost:3000 \
-    -d '[{"op": "replace", "path": "/environments/production/on", "value": true}]' \
-    --project default \
-    --flag test-flag
-```
+![demo](ldcli-setup.gif)
```
Comment (k3llymariee) [inline README.md:11]: placeholder for now but thought it was a nice touch

**Review by dbolson — APPROVED** [review]
This is great

File: README.md [hunk 2]
```diff
@@ -1,37 +1,69 @@
-# ldcli
+# LaunchDarkly CLI
 
-## Commands
+The LaunchDarkly CLI helps you manage your feature flags wherever you are, whether that's in the webapp, your terminal, or your IDE.
```
Comment (sunnyguduru) [inline README.md:3]: Are we going to have webapp features in the cli?

Comment (k3llymariee) [reply README.md:3]: this is for sure just placeholder text but this is just like a marketing-type phrase, "you can manage LD in the UI/website AND the CLI"

**Pull request merged**